butterfly-cli/build.gradle

```diff
@@ -7,6 +7,9 @@ dependencies {
     compile project(':butterfly-api')
     compile project(':butterfly-extensions-api')
     runtime project(':butterfly-core')
+//  TODO  remove these two after testing
```

Comment:
Remove these three lines

Reply:
Please remove these two dependencies from this PR (you an keep these changes in your laptop for your tests, just don't push them).